### PR TITLE
Add the tests for user stories 10 and 11

### PR DESCRIPTION
**Pull request by adumortier**
- added the tests for user stories 10 and 11

### CODE REVIEW

**Review by cgaddis36 — APPROVED** [review]
Good looking test, I believe we should check that the current user does exist like you suggested on line 37

File: spec/features/users/users_spec.rb [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+  it "returns a flash message if some fields are missing in the registration form" do
+    
+    visit "/items" # to access a page with the nav bar
+
+    click_on "Register"
+
+    expect(current_path).to eq("/register")
+
+    name = "George Costanza"
```
Comment (TravisBorgsmiller) [inline spec/features/users/users_spec.rb:50]: Great name

File: spec/features/users/users_spec.rb [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    fill_in :email, with: email
+    fill_in :password, with: password
+    fill_in :password_confirmation, with: password
+
+    click_on "Create User"
+
+    expect(current_path).to eq("/register")
+
+    expect(page).to have_content("Street address can't be blank and City can't be blank")
+    # need to add a test for which the address entered is already in the user table.
```
Comment (TravisBorgsmiller) [inline spec/features/users/users_spec.rb:72]: Good stuff.

**Review by TravisBorgsmiller — APPROVED** [review]

**Review by KellyIB — APPROVED** [review]
Tests looks great!

**Pull request merged**